docs/source/checkpoints/intro.rst

```diff
@@ -20,6 +20,26 @@ With sharded model weights, you can save and load the state of your training scr
 
 NeMo supports the distributed (sharded) checkpoint format from Megatron-Core. In Megatron-Core, it supports two backends: Zarr-based and PyTorch-based.
```

Comment:
NeMo supports the distributed (sharded) checkpoint format from Megatron Core. In Megatron Core, it supports two backends: Zarr-based and PyTorch-based.

Reply:
@yaoyu-33 that would be sth for you to account for in the destination branch `yuya/add_checkpoints_section`